gettsim/tests/test_docs.py

```diff
@@ -13,8 +13,7 @@ def test_funcs_in_doc_module_and_func_from_internal_files_are_the_same():
         internal_function_files, allow_imported_members=True
     )
 
-    # Fix because private functions are not imported in functions.py. Remove when all
-    # functions in the DAG became public.
+    # Private functions are not imported in functions.py.
```

Comment:
I would actually leave as is. I do not think we want to have all those comparisons on the website. Perfectly fine to follow Python conventions here (private stuff is visible, but should not be touched).

Reply:
No objections.